src/views/main/components/Summary.js

```diff
@@ -0,0 +1,360 @@
+        <Cloud2
+          className={styles.foregroundCloud}
+          fill={color}
+          style={{
+            "animation-delay": "-" + 5 + "s",
+            "animation-duration": 30 + "s",
+          }}
+        />
+        <Cloud3
+          className={styles.foregroundCloud}
+          fill={color}
+          style={{
+            "animation-delay": "-" + 10 + "s",
+            "animation-duration": 30 + "s",
+          }}
+        />
+        <Cloud1
+          className={styles.foregroundCloud}
+          fill={color}
+          style={{
+            "animation-delay": "-" + 15 + "s",
+            "animation-duration": 30 + "s",
+          }}
+        />
+        <Cloud2
+          className={styles.foregroundCloud}
+          fill={color}
+          style={{
+            "animation-delay": "-" + 20 + "s",
+            "animation-duration": 30 + "s",
+          }}
+        />
+        <Cloud3
+          className={styles.foregroundCloud}
+          fill={color}
+          style={{
+            "animation-delay": "-" + 25 + "s",
+            "animation-duration": 30 + "s",
+          }}
+        />
+      </>
+    );
+  }
+
+  renderBackgroundClouds() {
+    let color = "";
+    switch (this.getAirQualityLevel()) {
+      case -1:
+        color = "#4e1b16ff";
+        break;
+      case 1:
+        color = "#4e1b16ff";
+        break;
+      case 2:
+        color = "#988a57ff";
+        break;
+      case 3:
+        color = "#bbbd7bff";
+        break;
+      case 4:
+        color = "#a9dae1ff";
+        break;
+      case 5:
+        color = "#b7ddfcff";
+        break;
+    }
+    return this.getBackgroundClouds(9, color);
+  }
+
+  renderSmile() {
+    return this.getSmile(this.getAirQualityLevel());
+  }
+
+  renderRandomPerson() {
+    if (this.getAirQualityLevel() == -1) {
+      return <PersonGrave className={styles.person}></PersonGrave>;
+    }
+    return (
+      <>
+        {this.getPerson(this.state.personNum)}
+        {this.renderSmile()}
+        {this.getPersonForegeound(this.state.personNum)}
+      </>
+    );
+  }
+
+  renderForegroundClouds() {
+    let color = "";
+    switch (this.getAirQualityLevel()) {
+      case -1:
+        color = "#00000033";
+        break;
+      case 1:
+        color = "#00000033";
+        break;
+      case 2:
+        color = "#00000022";
+        break;
+      case 3:
+        color = "#00000011";
+        break;
+      case 4:
+        color = "#00000000";
+        break;
+      case 5:
+        color = "#00000000";
+        break;
+    }
+    return this.getForegroundClouds(color);
+  }
+
+  getDescriptionText() {
+    switch (this.getAirQualityLevel()) {
+      case -1:
+        return "You better stay in home!";
+      case 5:
+        return "Excellent air!";
+      case 4:
+        return "Good air quality!";
+      case 3:
+        return "Mediocre air quality";
+      case 2:
+        return "Poor air quality";
+      case 1:
+        return "Bad air quality";
```

Comment:
Are we able to change `switch` to `dict`? What are you think about this solution?

Reply:
yeah, it will take less space 👍